accounts/tests.py

```diff
@@ -169,78 +172,182 @@ def test_failure_post_with_password_similar_to_username(self):
-             'username':'sample',
-             'password1':'1111111111',
-             'password2':'1111111111',
+            'email': 'example@example.com',
+             'username': 'sample',
+             'password1': '1111111111',
+             'password2': '1111111111',
         }
-        reponse_only_numbers_password = self.client.post(reverse('signup'), data_only_numbers_password)
+        reponse_only_numbers_password = self.client.post(reverse('accounts:signup'), data_only_numbers_password)
         self.assertEquals(reponse_only_numbers_password.status_code, 200)
         self.assertFalse(User.objects.exists())
         
         self.assertFormError(reponse_only_numbers_password, 'form', 'password2', 'このパスワードは一般的すぎます。',  'このパスワードは数字しか使われていません。')
         
-       
+
     def test_failure_post_with_mismatch_password(self): 
         #パスワード１と２が異なるデータでリクエストを送信する
         data_mismatch_password = {
-            'email':'example@example.com',
-             'username':'sample',
-             'password1':'example12345',
-             'password2':'example123456',
+            'email': 'example@example.com',
+             'username': 'sample',
+             'password1': 'example12345',
+             'password2': 'example123456',
         }
         
-        response_mismatch_password = self.client.post(reverse('signup'), data_mismatch_password)
+        response_mismatch_password = self.client.post(reverse('accounts:signup'), data_mismatch_password)
         self.assertFalse(User.objects.exists())
         self.assertEquals(response_mismatch_password.status_code, 200)
-
         self.assertFormError(response_mismatch_password, 'form', 'password2', '確認用パスワードが一致しません。')
 
 
 class TestHomeView(TestCase):
+    def setUp(self):
+        data = {
+            'username': 'yamada',
+            'email': 'asaka@test.com',
+            'password1': 'wasurenaide1108',
+            'password2': 'wasurenaide1108',
+        }
+        self.client.post(reverse('accounts:signup'), data)
+        
+
     def test_success_get(self):
-        response = self.client.get(reverse('home'))
+        response = self.client.get(reverse('accounts:home'))
         self.assertEquals(response.status_code, 200)
         self.assertTemplateUsed(response, 'accounts/home.html')
 
 
 class TestLoginView(TestCase):
+   
     def test_success_get(self):
-        pass
+        response_get = self.client.get(reverse('accounts:login'))
+        self.assertEquals(response_get.status_code, 200)
+        self.assertTemplateUsed(response_get, 'accounts/login.html')
 
     def test_success_post(self):
-        pass
+        User.objects.create_user(username='yamada', email='asaka@test.com', password='wasurenaide1108')
+        data_post = {
+            'username':'yamada',
+            'password':'wasurenaide1108',
+        }
+       
+        self.url = reverse('accounts:login')
+        response_post = self.client.post(reverse('accounts:login'), data_post)

```

Comment:
```suggestion
        response_post = self.client.post(self.url, data_post)
```
変数に代入したならこのようにしておきましょう。
それと、やはり、TestLoginViewで
```
def setUp(self):
    User.objects.create_user(username='yamada', email='asaka@test.com', password='wasurenaide1108')
    self.url = reverse('accounts:login')
```
としておいた方がいいと思います。